version

```diff
@@ -1 +1 @@
-v1
+v1.1
```

Comment:
suggestion: There's no really good way to handle this right now. I, at least, recommend that we convert to using the tag versioning approach that is used by our services. The issue I'm considering is that moving this to `v1.1` is equivalent to `v2.0` because `v1` customers will stop receiving updates. If we replicated our other builds then we'd have a `latest`, `v1`, and `v1.1.0` build.

However, none of that will address your original intent which was to handle this as a progressive rollout. To that end, could the `v1.1` tag/branch be temporary and have this revert to `v1` once we're confident we've resolved any issues?

Reply:
`... v1 customers will stop receiving updates`
Yes this is my intent by updating the version number to `v1.1` so consumers who don't want to adopt the latest changes can opt into it at a later time. 

The problem with not updating the version number here is as soon as we merge, all consumers of `v1` will get this update immediately and we still have one known repository that doesn't work great with `GOPROXY`.

Reply:
My plan for gradual roll out is to have every repository that uses sdcli point to `v1.1`, consumers who can't adopt these changes will still be on `v1` but we should fix them so they don't get "left behind".

Reply:
> have every repository that uses sdcli point to v1.1
That's my primary concern. That indicates that this is a breaking change is actually a `v2` release. I'm OK using a major version if we plan to force upgrades to get new updates. Alternatively, I'd suggest we schedule a date on which we switch this file back to `v1` and "release" the experimental changes to `v1` stable branch users. As an added alternative, we might consider using `GOPROXY=https://proxy.golang.org,direct` in order to provide a fallback for problematic modules.

Reply:
Or @kconwayatlassian should we just create an older tagged version of sdcli without this change, then we keep this change at `v1` so everyone gets the latest version and if anything breaks they can roll back to an older tagged version I could create without this change?

Reply:
I doubt we have folks outside of our team using this right now and I recognize the somewhat academic nature of this thread. If we had customers, I'd feel that giving folks a rollback point is pushing too much effort and risk onto the customer. I do like the idea of having a branch build where we can incubate the change before rolling it out. I think the sticking point is how customers receive the update after we've finished incubating.

If customers need to adjust version numbers then I'd recommend calling this `v2` to clearly indicate that a forced upgrade is required to get updates. If we want to roll it out under `v1` then I'd recommend only changing a few of our systems to the new branch build and then switching the version back to `v1` when we're ready to roll out.

Reply:
Roger that, we will test out `v1.1` with a few repositories and once we get more positive results, we'll roll back the tagged version to `v1` for master. I don't expect this to break many repositories honestly I'm just being cautious.